### ISSUE TITLE
Translation of Special Terms 

### ISSUE THREAD

**Issue by negarjf** [vuejs-fa/fa.vuejs.org#19]
In a context like this Vue document, the final purpose is to make developers able to use the framework. Since the framework terms are all English words I personally believe that terms like `directive`, `component`, `props`, etc. shouldn't be translated into Persian.

All in all, we need to follow the same pattern in our translations, so before going any further I think we should make a decision.

For instance, how do you prefer to translate this sentence:

> Props are custom attributes you can register on a component.

**Comment by payamnj** [comment]
I completely agree. What do you think about having a **.md** file in the repo for translators reference, something like this:
<table style="width:100%">
  <tr>
    <th>Original term</th>
    <th>In translated content</th>
  </tr>
  <tr>
    <td>props</td>
    <td>props</td>
  </tr>
  <tr>
    <td>component</td>
    <td>component</td>
  </tr>
  <tr>
    <td>library</td>
    <td>لایبری</td>
  </tr>
  <tr>
    <td>...</td>
    <td>...</td>
  </tr>
</table>

Or if it's not a hassle, create a component tag like `<fa :term='props'>`.  then we will have all of them in one place.

**Comment by negarjf** [comment]
I think using component tags between Persian text would be problematic, but I have already created a Wiki page for [glossary](https://github.com/vuejs-fa/fa.vuejs.org/wiki/Glossary) and list all the terms I have found in existing translated pages.

We can enhance the structure and translations over time.
